introduction.md

```diff
@@ -1,3 +1,9 @@
-##Hello everyone, did you know it's chucklesoclock?
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?